deploying/docker_install.md

```diff
@@ -0,0 +1,72 @@
+# Installation and configuration of Docker
+## Ubuntu 15.04:
+1. Verify current kernel version by typing:
+```bash
+uname -r
+```
+At the time of this writing, the current kernel version is 3.19.0-27
+
+2. Ensure that required and optional packages are installed:
+```bash
+sudo apt-get install linux-image-3.19.0-27-generic linux-image-extra-3.19.0-27-generic
```

Comment:
This seems a little aggressive. perhaps suggest that the user should have at least this image version?  I have 3.19.0-25 on my laptop and docker works fine